frontend/src/core/codemirror/completion/hints.ts

```diff
@@ -116,11 +116,6 @@ const cursorTooltipField = StateField.define<Tooltip[]>({
     return [];
   },
   update(tooltips, tr) {
-    // If the document or selection has changed, clear the tooltips
-    if (tr.docChanged || tr.selection) {
-      return [];
-    }
-
```

Comment:
Maybe this is needed sometimes? I see completion overlapping with tooltip now. See video below

[multiple-completions-tooltips.webm](https://github.com/marimo-team/marimo/assets/1994308/29b1a328-507f-4b4d-8f44-92ae4a6a1d44)

Reply:
i think i can clear this in a different place to avoid the overlap